brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Star Rating - no need of General panel tab under General tab, show setting directly.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/07524229-4e34-4b01-80cb-950bb5a9013e?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

- https://share.getcloudapp.com/o0uRnowr

### Types of changes




- Improvement

### How has this been tested?

- Drag and drop Star Rating
- Check General Tab. No panel tab should be present. Setting displayed directly.
- See screencast - https://share.getcloudapp.com/o0uRnowr
- Check Star rating on responsive devices.

### Checklist:
